src/bin/mosaic-libraries.ts

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+import * as commander from 'commander';
+
+import { version } from '../../package.json';
```

Comment:
All references to `package.json` have to be removed. See #75 ("version and description References")

Reply:
OK, I will make this change if #75 is merged before.  👼

Reply:
You have to remove it either way, the change does not depend on #75.

Reply:
Done